src/scripts/controller/apiServices.ts

```diff
@@ -0,0 +1,23 @@
+import { Product, ProductResponse } from '../model/interfaces';
+
+export class ApiServices {
+    private URL: string = 'https://dummyjson.com/products';
```

Comment:
Minor Nit: can make it `readonly` to ensure no one can modify it.
```suggestion
    private readonly URL: string = 'https://dummyjson.com/products';
```

Reply:
Added `readonly` (https://github.com/krushna-simform/product-management-system/pull/3/commits/210e958ec1173b5ae54ea2e536a91d7abaea3d27)